packages/documentation/copy/en/release-notes/TypeScript 1.6.md

```diff
@@ -439,7 +439,7 @@ if (isCat(x)) {
 }
 ```
 
-## `exclude` property support in tsconfig.json
+## [`exclude`](/tsconfig#exclude) property support in tsconfig.json
```

Comment:
```suggestion
## `exclude` property support in tsconfig.json
```

This is a heading and cannot contain a link.

Reply:
As in, the renderer doesn't support links in headings? Markdown itself does support that

If so a lint check for that would be good to add to the automation

Reply:
Maybe it can, but I don't think we have any examples of that. That and sometimes headings are clickable themselves and this would conflict. I don't think it's worth it for this heading.